WcaOnRails/app/views/contacts/fix_personal_information.html.erb

```diff
@@ -0,0 +1,31 @@
+<% provide(:title, t('.title')) %>
+<div class="container">
+  <h1><%= yield(:title) %></h1>
+
+  <p><%= t('.description') %></p>
+
+  <%= simple_form_for @contact, url: contact_fix_personal_information_path, html: { multipart: true } do |f| %>
+    <% if current_user %>
+      <%= f.input :your_email %>
+    <% else %>
+      <%= f.hidden_field :your_email %>
+    <% end %>
+
+    <%= f.input :wca_id, as: :user_ids, persons_table: true, only_one: true %>
+
+    <div id="person-fields">
+      <%= f.input :name %>
+      <%= f.input :gender, collection: User::ALLOWABLE_GENDERS, label_method: lambda { |g| I18n.t("enums.user.gender.#{g}") } %>
+      <%= f.input :dob, as: :date_picker %>
+
+      <strong><%= t('.important_message_html') %></strong>
+
+      <%= f.input :document, as: :file %>
+      <%= recaptcha_tags %>
+      <div>
+        </br>
```

Comment:
This is not a valid tag. I see we had it like that for a long time ^^

Reply:
Wowza. Changed to `<br>`!